sources/10MBDRIVER

```diff
@@ -74,47 +73,41 @@ Copyright (c) 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1990, 1991 by Venue & Xe
              PACKET])
 
 (\10MBSENDPACKET
-  [LAMBDA (NDB PACKET)                                   (* ; "Edited 14-Apr-88 15:30 by bvm")
-    (PROG ([DROPIT (AND \ETHERLIGHTNING (EQ 0 (RAND 0 \ETHERLIGHTNING]
-           IOCB BUFLENGTH)
+  [LAMBDA (NDB PACKET)                                       (* ; "Edited 31-Oct-89 14:10 by bvm")
+    (PROG NIL
           [COND
              (\RAWTRACING (\MAYBEPRINTPACKET PACKET 'RAWPUT]
           [COND
-             ([AND (NOT (fetch NDBCANHEARSELF of NDB))
```

Comment:
Note that Dodo isn't the ONLY way to send and receive ethernet packets -- it still matters whether the "real" ethernet interface (/dev/nit, /dev/bpf, DLPI) "can hear itself".